doc/docportal/advanced_topics/configuration_file.rst

```diff
@@ -363,6 +364,17 @@ There are many recognized configuration keys. In the table below, each key is ei
 	- 2gs
 	- atari
 	- macintosh "
+		":ref:`retrowaveopl3_bus <adlib>`",string,,"
+	Specifies how the RetroWave OPL3 is connected:
+	- serial (connected to a USB port using a PotatoPi)
+	- spi (connected as a HAT using SPI) "
+		":ref:`retrowaveopl3_disable_buffer <adlib>`",boolean,false,
+		":ref:`retrowaveopl3_port <adlib>`",string,,"
+	Specifies the serial port that the RetroWave OPL3 is connected to.
+	For example:
+	- COM3
+	- ttyACM2 "
+		":ref:`retrowaveopl3_spi_cs <adlib>`",string,,Specifies the GPIO chip and line that the RetroWave OPL3 is connected to. Use the format <chip>,<line>.
```

Comment:
Add quotation marks to the description “Specifies the GPIO chip…” to fix the formatting in the table - at the moment it’s reading the comma and moving ```<line>``` to a new cell